build/enforcer/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Hibernate Search, full-text search for your domain model
+ ~
+ ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.hibernate.search</groupId>
+        <artifactId>hibernate-search-parent</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>hibernate-search-build-enforcer</artifactId>
+
+    <name>Hibernate Search Build - Enforcer Rules</name>
+    <description>Hibernate Search custom enforcer rules</description>
+
+    <properties>
+
+        <!--
+            We want to skip these checks since this module is built before the build-config,
+            as a result the required files to run the checks will be missing.
+         -->
+        <checkstyle.skip>true</checkstyle.skip>
+        <forbiddenapis.skip>true</forbiddenapis.skip>
+        <format.skip>true</format.skip>
+
+        <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+        <jqassistant.skip>true</jqassistant.skip>
+
+        <version.sisu-maven-plugin>0.9.0.M1</version.sisu-maven-plugin>
```

Comment:
Maybe put this in the root POM, with plugin management just for the version? We've managed to put plugin versions in a central place so far, better stick to that.